spec/javascript/e2e/SupplementalApplicationPage.e2e.js

```diff
@@ -39,7 +39,140 @@ describe('SupplementalApplicationPage', () => {
 
     await supplementalApplicationSteps.testStatusModalUpdate(page)
 
-    browser.close()
+    await browser.close()
+  }, DEFAULT_E2E_TIME_OUT)
+
+  test('should allow a new rental assistance to be created', async () => {
+    let browser = await puppeteer.launch({ headless: HEADLESS })
+    let page = await browser.newPage()
+
+    await sharedSteps.loginAsAgent(page)
+    await sharedSteps.goto(page, `/applications/${LEASE_UP_LISTING_APPLICATION_ID}/supplementals`)
+
+    // Click on the Add Rental Assistance button in the Rental Assistance section
+    await page.click('button#add-rental-assistance')
+
+    // Wait for the new rental assistance form to appear
+    await page.waitForSelector('.rental-assistance-new-form')
+
+    // Set up the values we'll use to fill out the rental assistance form
+    const [typeVal, typeLabel] = await page.$eval(
+      '.rental-assistance-new-form .rental-assistance-type option:nth-child(2)',
+      e => [e.value, e.textContent]
+    )
+    const recurring = 'Yes'
```

Comment:
nit: this section might be bit clearer if the page.$evals are grouped before or after the recurring/amount.

Reply:
i wanted to keep them in the order that they're filled in on the page - especially for debugging, it makes it a lot easier to see "ok, we wanted to use this value for this field, then this value for the next....and the test got to this field, then the next..."